src/assets/wise5/services/moveNodesService.ts

```diff
@@ -200,7 +200,46 @@ export class MoveNodesService {
   }
 
   private moveInactiveNodeToInactiveSection(node: any, nodeIdToInsertAfter: string): void {
-    this.projectService.removeNodeFromInactiveNodes(node.id);
+    this.removeNodeFromInactiveNodes(node.id);
     this.projectService.addInactiveNodeInsertAfter(node, nodeIdToInsertAfter);
   }
+
+  private removeNodeFromInactiveNodes(nodeId: string): void {
+    let node = null;
+    let parentGroup = this.projectService.getParentGroup(nodeId);
+    if (parentGroup != null) {
+      this.projectService.removeChildFromParent(nodeId);
+    }
+
+    let inactiveNodes = this.projectService.project.inactiveNodes;
```

Comment:
We can change the let to const.

```suggestion
    const inactiveNodes = this.projectService.project.inactiveNodes;
```